rotkehlchen/tests/unit/decoders/test_gearbox.py

```diff
@@ -0,0 +1,264 @@
+        ethereum_inquirer: 'EthereumInquirer',
+        ethereum_accounts: list['ChecksumEvmAddress'],
+):
+    ethereum_inquirer.ensure_cache_data_is_updated(
+        cache_type=CacheType.GEARBOX_POOL_ADDRESS,
+        query_method=query_gearbox_data,
+        save_method=save_gearbox_data_to_cache,
+    )
+    tx_hash = deserialize_evm_tx_hash('0x92d178bbe5152cad47029b0c130450848ee46084e72addd09ba955631af1325b')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=ethereum_inquirer,
+        database=database,
+        tx_hash=tx_hash,
+    )
+    timestamp, gas, deposit_amount, lp_token_amount = TimestampMS(1716899027000), '0.004946659515956316', '6500000', '6147276.510091'  # noqa: E501
+    expected_events = [
+        EvmEvent(
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(gas)),
+            location_label=ethereum_accounts[0],
+            notes=f'Burned {gas} ETH for gas',
+            tx_hash=tx_hash,
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            sequence_index=154,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.INFORMATIONAL,
+            event_subtype=HistoryEventSubType.APPROVE,
+            asset=A_USDC,
+            balance=Balance(amount=FVal(6500000)),
+            location_label=ethereum_accounts[0],
+            notes=f'Set USDC spending approval of {ethereum_accounts[0]} by 0x53D5BD0E7fAa9ee3eafEf7C5572D54DB1b7f5b25 to {deposit_amount}',  # noqa: E501
+            tx_hash=tx_hash,
+            address=string_to_evm_address('0x53D5BD0E7fAa9ee3eafEf7C5572D54DB1b7f5b25'),
+        ), EvmEvent(
+            sequence_index=155,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.DEPOSIT,
+            event_subtype=HistoryEventSubType.DEPOSIT_ASSET,
+            asset=A_USDC,
+            balance=Balance(amount=FVal(deposit_amount)),
+            location_label=ethereum_accounts[0],
+            notes=f'Deposit {deposit_amount} USDC to Gearbox',
+            tx_hash=tx_hash,
+            counterparty=CPT_GEARBOX,
+            address=string_to_evm_address('0x53D5BD0E7fAa9ee3eafEf7C5572D54DB1b7f5b25'),
+        ), EvmEvent(
+            sequence_index=162,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.RECEIVE,
+            event_subtype=HistoryEventSubType.RECEIVE_WRAPPED,
+            asset=Asset('eip155:1/erc20:0x9ef444a6d7F4A5adcd68FD5329aA5240C90E14d2'),
+            balance=Balance(amount=FVal(lp_token_amount)),
+            location_label=ethereum_accounts[0],
+            notes=f'Receive {lp_token_amount} farmdUSDCV3 after staking in Gearbox',
```

Comment:
not an error in the test but can you change the note here. It's not staking but depositing. Gearbox is a lending protocol.

```suggestion
            notes=f'Receive {lp_token_amount} farmdUSDCV3 after staking in Gearbox',
```